_extras/extra_exercises.md

```diff
@@ -44,7 +44,8 @@ or not (yet) added to the main lesson.
 > >
 > > In the first case we used a temporary variable `temp` to keep the value of `left` before we
 > > overwrite it with the value of `right`. In the second case, `right` and `left` are packed into a
-> > list and then unpacked into `left` and `right`.
+> > [tuple]({{ page.root }}/reference.html#tuple)
```

Comment:
I changed `list` to `tuple` and added a link to the `reference.html` `tuple` entry.